src/Uri.php

```diff
@@ -265,6 +265,12 @@ public static function isSameDocumentReference(UriInterface $uri, UriInterface $
      */
     public static function removeDotSegments($path)
     {
+        @trigger_error(
```

Comment:
The `@` sign mutes the warning, so that it will never be visible to the implementing developer.